core/src/main/java/com/linecorp/armeria/common/logging/JsonLogFormatterBuilder.java

```diff
@@ -0,0 +1,128 @@
+
+    @Nullable
+    private ObjectMapper objectMapper;
+
+    JsonLogFormatterBuilder() {}
+
+    /**
+     * Sets the {@link ObjectMapper} that will be used to convert an object into a JSON format message.
+     */
+    public JsonLogFormatterBuilder objectMapper(ObjectMapper objectMapper) {
+        this.objectMapper = requireNonNull(objectMapper, "objectMapper");
+        return this;
+    }
+
+    @Override
+    public JsonLogFormatterBuilder requestHeadersSanitizer(
+            BiFunction<? super RequestContext, ? super HttpHeaders, ? extends JsonNode>
+                    requestHeadersSanitizer) {
+        return (JsonLogFormatterBuilder) super.requestHeadersSanitizer(requestHeadersSanitizer);
+    }
+
+    @Override
+    public JsonLogFormatterBuilder responseHeadersSanitizer(
+            BiFunction<? super RequestContext, ? super HttpHeaders, ? extends JsonNode>
+                    responseHeadersSanitizer) {
+        return (JsonLogFormatterBuilder) super.responseHeadersSanitizer(responseHeadersSanitizer);
+    }
+
+    @Override
+    public JsonLogFormatterBuilder requestTrailersSanitizer(
+            BiFunction<? super RequestContext, ? super HttpHeaders, ? extends JsonNode>
+                    requestTrailersSanitizer) {
+        return (JsonLogFormatterBuilder) super.requestTrailersSanitizer(requestTrailersSanitizer);
+    }
+
+    @Override
+    public JsonLogFormatterBuilder responseTrailersSanitizer(
+            BiFunction<? super RequestContext, ? super HttpHeaders, ? extends JsonNode>
+                    responseTrailersSanitizer) {
+        return (JsonLogFormatterBuilder) super.responseTrailersSanitizer(responseTrailersSanitizer);
+    }
+
+    @Override
+    public JsonLogFormatterBuilder headersSanitizer(
+            BiFunction<? super RequestContext, ? super HttpHeaders, ? extends JsonNode> headersSanitizer) {
+        return (JsonLogFormatterBuilder) super.headersSanitizer(headersSanitizer);
+    }
+
+    @Override
+    public JsonLogFormatterBuilder requestContentSanitizer(
+            BiFunction<? super RequestContext, Object, ? extends JsonNode> requestContentSanitizer) {
+        return (JsonLogFormatterBuilder) super.requestContentSanitizer(requestContentSanitizer);
+    }
+
+    @Override
+    public JsonLogFormatterBuilder responseContentSanitizer(
+            BiFunction<? super RequestContext, Object, ? extends JsonNode> responseContentSanitizer) {
+        return (JsonLogFormatterBuilder) super.responseContentSanitizer(responseContentSanitizer);
+    }
+
+    @Override
+    public JsonLogFormatterBuilder contentSanitizer(
+            BiFunction<? super RequestContext, Object, ? extends JsonNode> contentSanitizer) {
+        return (JsonLogFormatterBuilder) super.contentSanitizer(contentSanitizer);
+    }
+
+    /**
+     * Returns a newly-created {@link JsonLogFormatter} based on the properties of this builder.
```

Comment:
Since `JsonLogFormatter` is package private

```suggestion
     * Returns a newly-created {@link LogFormatter} based on the properties of this builder.
```